scripts/shared/clusters.sh

```diff
@@ -29,6 +46,7 @@ eval set -- "${FLAGS_ARGV}"
 k8s_version="${FLAGS_k8s_version}"
 olm_version="${FLAGS_olm_version}"
 [[ -z "${k8s_version}" ]] && k8s_version="${DEFAULT_K8S_VERSION}"
+kind="${kind_kind_binaries[$k8s_version]}"
```

Comment:
```suggestion
kind="${kind_kind_binaries[$k8s_version]:-kind}"
```

Wouldn't this work, thus obviating the need to declare kind binaries for 1.17-1.23?

Also it would fit well with 
> kind-0.12 is the default

Reply:
Yes, the last change was a quick fix to see if I could get the whole set of jobs passing.